app/apollo/utils/versionUtils.js

```diff
@@ -295,13 +295,14 @@ const validateNewVersions = async ( org_id, { channel, newVersions }, context )
   if( !newVersions || newVersions.length == 0 ) return;
 
   const { models } = context;
-  const total = await models.DeployableVersion.count( { org_id, channel_id: channel.uuid } );
+  const deployableVersionObjs = await models.DeployableVersion.find( { org_id, channel_id: channel.uuid } );
```

Comment:
Is `find` equally efficient as `count`?

Reply:
Likely not -- at the very least, `find` will use more bandwidth to return the content.  However `validateNewVersions` is infrequently called (only when creating new Versions) and the impact is therefore acceptable.